src/app/search/search-filters/sockets.tsx

```diff
@@ -290,6 +290,20 @@ const socketFilters: FilterDefinition[] = [
       (item: DimItem) =>
         item.sockets && compare!(countEnhancedPerks(item.sockets)),
   },
+  {
+    keywords: 'discontinuedperk',
```

Comment:
The linked issue suggested `retiredperk`, which I prefer. Thoughts?

Reply:
I agree, I was not a huge fan of discontinued. Nice I like retired much better.